Mobl_ios_Callisto-master/MealTrackingPilot/Extensions/UIView+Constraint.swift

```diff
@@ -0,0 +1,258 @@
+//
+//  UIView+Extensions.swift
+//  MealTrackingPilot
+//
+//  Created by Max Litteral on 3/24/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+typealias ConstraintDictionary = [String : NSLayoutConstraint]
+
+let IPConstraintKeyTop = "IPConstraintKeyTop"
+let IPConstraintKeyLeft = "IPConstraintKeyLeft"
+let IPConstraintKeyBottom = "IPConstraintKeyBottom"
+let IPConstraintKeyRight = "IPConstraintKeyRight"
+
+extension UIView {
+
+    func constrainView(_ view: UIView?, top: CGFloat, left: CGFloat, bottom: CGFloat, right: CGFloat) -> ConstraintDictionary? {
+        return constrainView(view, to: UIEdgeInsets(top: top, left: left, bottom: bottom, right: right))
+    }
+    
+    func constrainView(_ view: UIView?, to insets: UIEdgeInsets) -> ConstraintDictionary? {
+        var constraints: ConstraintDictionary = [:]
+        if insets.top != CGFloat(NSNotFound) {
```

Comment:
This is not correct way check the insets values in Swift. Can you please check how we will compare this values